app/models/user.rb

```diff
@@ -421,6 +421,16 @@ def relationships
     @relationships ||= get_relationships_array
   end
 
+  def identity_serial
+    {
+      'uuid' => uuid,
```

Comment:
If we call MPI to get SSN, might as well call it to get the rest of the PII data as well